scripts/develop

```diff
@@ -10,11 +10,5 @@ if [[ ! -d "${PWD}/config" ]]; then
     hass --config "${PWD}/config" --script ensure_config
 fi
 
-# Set the path to custom_components
-## This let's us have the structure we want <root>/custom_components/integration_blueprint
-## while at the same time have Home Assistant configuration inside <root>/config
-## without resulting to symlinks.
-export PYTHONPATH="${PYTHONPATH}:${PWD}/custom_components"
```

Comment:
Removing this makes it only work in a devcontainer.
This repo exist as a starting point, not to catch every possible edge case out there.

If you want to showcase that, this repo is better.
<https://github.com/home-assistant/example-custom-config>

If you just want to share how you do things, there is this section on the forums:
<https://community.home-assistant.io/c/community-guides/51>